extensions/blocks/podcast-player/components/podcast-player.js

```diff
@@ -132,14 +132,14 @@ export class PodcastPlayer extends Component {
 	setAudioSource = noop;
 
 	render() {
-		const { tracks, itemsToShow } = this.props;
+		const { id, tracks, itemsToShow } = this.props;
 		const { playerState, currentTrack } = this.state;
 
 		const tracksToDisplay = tracks.slice( 0, itemsToShow );
 
 		return (
 			<div className={ playerState }>
-				<Header track={ this.getTrack( currentTrack ) }>
+				<Header id={ id } track={ this.getTrack( currentTrack ) }>
```

Comment:
Should I use `playerId={ id }` instead of  `id={ id }` here?

Reply:
good idea to prevent a potential confusion with html's `id` 👍

otherwise, it might look like we set the same id on multiple elements